lib/validation.tsx

```diff
@@ -202,12 +202,29 @@ export const validateOnSubmit = (values: FormValues, props: DynamicFormProps): R
  * @param props
  */
 export const getErrorList = (
-  props: InnerFormProps & FormikProps<FormValues>
+  props: InnerFormProps & FormikProps<FormValues> & DynamicFormProps
 ): JSX.Element | null => {
+  if (!props.formConfig) {
+    return null;
+  }
   let errorList;
-  const formElementErrors = Object.entries(props.errors);
-  if (props.touched && formElementErrors.length) {
-    errorList = formElementErrors.map(([formElementKey, formElementErrorValue]) => {
+  const formElementErrors: [string, string | undefined][] = Object.entries(props.errors);
+  const layout: string[] = props.formConfig.layout;
+  if (formElementErrors.length === 0 || layout.length === 0) {
+    return null;
+  }
+
+  const sortedFormElementErrors: [string, string | undefined][] = [];
+  layout.map((element) => {
```

Comment:
This way you just need to do the following 

```js
const sortedFormElementErrors = layout.map((element) => {
    return [ element, props.errors[element] ]
})
```

Reply:
Had to add in a filter as well, otherwise it returned an array that was the full size of `layout`.